### PR TITLE
Mise à jour fr.json

### PR DESCRIPTION
**Pull request by Dany-LF**
Corrected translation.

### CODE REVIEW

File: translations/fr.json [hunk 1]
```diff
@@ -57,13 +57,13 @@
...
-        "errorValidationAdminRoleNotAssigned": "Veuillez attribuer le rôle guildcopy au bot",
+        "errorValidationAdminRolePosition": "Le rôle `guildcopy` doit être le rôle le plus haut sur le serveur cible.",
+        "errorValidationAdminRoleNotAssigned": "Veuillez attribuer le rôle `guildcopy` au bot",
         "errorRestoreNotExistent": "Restauration spécifiée mais la sauvegarde du serveur '[@@1@@]' n'existe pas",
-        "errorSerializationOriginalNotExistent": "Le serveur original à copier n'existe pas. Veuillez vérifier si l'identifiant dans les paramètres est correct et si le bot est également membre de ce serveur",
-        "errorSerializationNoBanPermissions": "Vous avez essayé de copier les bannissements sans donner au bot les autorisations BAN_MEMBERS sur le serveur d'origine",
+        "errorSerializationOriginalNotExistent": "Le serveur original à copier n'existe pas. Veuillez vérifier si l'identifiant dans les paramètres est correct et également si le bot est membre de ce serveur",
+        "errorSerializationNoBanPermissions": "Vous avez essayé de copier les bannissements sans donner au bot l'autorisations `BAN_MEMBERS` sur le serveur d'origine",
```
Comment (Androz2091) [inline translations/fr.json]:
```suggestion
        "errorSerializationNoBanPermissions": "Vous avez essayé de copier les bannissements sans donner au bot la permission `BAN_MEMBERS` sur le serveur d'origine",
```

File: translations/fr.json [hunk 2]
```diff
@@ -9,41 +9,41 @@
...
+        "messageCreatorVoiceDataDebug": "[@@1@@].[@@2@@] Salon vocal \"[@@3@@]\" créé(s)",
         "messageCreatorEmojiData": "[@@1@@]. Création des émojis",
-        "messageCreatorEmojiDataDebug": "[@@1@@].[@@2@@] Emoji \"[@@3@@]\" créé",
+        "messageCreatorEmojiDataDebug": "[@@1@@].[@@2@@] Emoji \"[@@3@@]\" créé(s)",
         "messageCreatorBanData": "[@@1@@]. Bannissements des utilisateurs",
         "messageCreatorBanDataDebug": "[@@1@@].[@@2@@] Utilisateur [@@3@@] banni",
-        "messageGuildCopyFinished": "Copie du serveur termonée! Le rôle `[@@1@@]` peut maintenant être supprimé",
-        "messageGuildCopyFinishedInvites": "Lien d'invitation pour tous les bots",
+        "messageGuildCopyFinished": "Copie du serveur terminée! Le rôle `[@@1@@]` peut maintenant être supprimé",
+        "messageGuildCopyFinishedInvites": "Liste des lien d'invitation pour tous les bots",
```
Comment (Androz2091) [inline translations/fr.json]:
```suggestion
        "messageGuildCopyFinishedInvites": "Liste des liens d'invitation pour tous les bots",
```

File: translations/fr.json [hunk 3]
```diff
@@ -9,41 +9,41 @@
...
-        "messageSerializerCategoryData": "[@@1@@]. Sérialisation des données des catégories",
+        "messageSerializerCategoryData": "[@@1@@]. Sérialisation des données de catégories",
         "messageSerializerCategoryDataDebug": "[@@1@@].1 [@@2@@] catégorie(s) séralisée(s)",
-        "messageSerializerTextData": "[@@1@@]. Sérialisation des données des salons textuels",
+        "messageSerializerTextData": "[@@1@@]. Sérialisation des données de salons textuels",
         "messageSerializerTextDataDebug": "[@@1@@].1 [@@2@@] salon(s) textuel(s) sérialisé(s)",
-        "messageSerializerVoiceData": "[@@1@@]. Sérialisation des données des salons vocaux",
-        "messageSerializerVoiceDataDebug": "[@@1@@].1 Serialized [@@2@@] salon(s) vocal(aux) sérialisé(s)",
+        "messageSerializerVoiceData": "[@@1@@]. Sérialisation des données de salons vocaux",
+        "messageSerializerVoiceDataDebug": "[@@1@@].1 Sérialisation [@@2@@] salon(s) vocal(aux) sérialisé(s)",
```
Comment (Androz2091) [inline translations/fr.json]: Here, it is better to keep 'des' rather than 'de'.

Comment (Androz2091) [inline translations/fr.json]: Why did you add "Sérialisation" here? It doesn't mean anything 🤔

File: translations/fr.json [hunk 4]
```diff
@@ -9,41 +9,41 @@
...
         "messageSerializerEmojiData": "[@@1@@]. Sérialisation des émojis",
         "messageSerializerEmojiDataDebug": "[@@1@@].1 [@@2@@] émoji(s) sérialisé(s)",
         "messageSerializerBanData": "[@@1@@]. Sérialisation des bannissements",
         "messageSerializerBanDataDebug": "[@@1@@].1 [@@2@@] bannissement(s) sérialisé(s)",
         "messageSerializerSave": "[@@1@@]. Sauvegarde des données du serveur dans un fichier",
         "messageSerializerSaveFinished": "[@@1@@]. Sérialisation terminée et données sauvegardées avec succès",
-        "messageBackupDone": "[@@1@@]. L'exécution du programme est terminée parce que la sauvegarde a été spécifiée",
+        "messageBackupDone": "[@@1@@]. L'exécution du programme est terminée parce-que la sauvegarde a été spécifiée",
```
Comment (Androz2091) [inline translations/fr.json]: "parce que" is not hyphenated.

File: translations/fr.json [hunk 5]
```diff
@@ -9,41 +9,41 @@
...
+        "messageCreatorCategoryDataDebug": "[@@1@@].[@@2@@] Catégorie \"[@@3@@]\" créée(s)",
         "messageCreatorTextData": "[@@1@@]. Création des salons textuels",
-        "messageCreatorTextDataDebug": "[@@1@@].[@@2@@] Salon textuel \"[@@3@@]\" créé",
+        "messageCreatorTextDataDebug": "[@@1@@].[@@2@@] Salon textuel \"[@@3@@]\" créé(s)",
         "messageCreatorVoiceData": "[@@1@@]. Création des salons vocaux",
-        "messageCreatorVoiceDataDebug": "[@@1@@].[@@2@@] Salon vocal \"[@@3@@]\" créé",
+        "messageCreatorVoiceDataDebug": "[@@1@@].[@@2@@] Salon vocal \"[@@3@@]\" créé(s)",
         "messageCreatorEmojiData": "[@@1@@]. Création des émojis",
-        "messageCreatorEmojiDataDebug": "[@@1@@].[@@2@@] Emoji \"[@@3@@]\" créé",
+        "messageCreatorEmojiDataDebug": "[@@1@@].[@@2@@] Emoji \"[@@3@@]\" créé(s)",
```
Comment (Androz2091) [inline translations/fr.json]: Here the "s" are useless, everything is in the singular.

**Comment by Androz2091** [comment]
Hmmm some of the reviews aren't resolved but are shown like this by Github

**Comment by Dany-LF** [comment]
I know

**Comment by Jisagi** [comment]
Are all the improvements done so i can pull it?

**Comment by Dany-LF** [comment]
Yes you can

**Comment by Jisagi** [comment]
Perfect. The update for the (finally released) v12 stable release of d.js will then follow at some point today as well.

**Comment by Dany-LF** [comment]
Nice, now this is perfect!
